<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MTG_DraftClass</class>
 <widget class="QMainWindow" name="MTG_DraftClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>866</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MTG_Draft</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>694</x>
      <y>740</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="cardImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>190</width>
      <height>265</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cardImage_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>190</width>
      <height>265</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cardImage_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>190</width>
      <height>265</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cardImage_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>190</width>
      <height>265</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cardImage_5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>270</y>
      <width>190</width>
      <height>265</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cardImage_6">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>270</y>
      <width>190</width>
      <height>265</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cardImage_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>190</width>
      <height>265</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cardImage_8">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>540</y>
      <width>190</width>
      <height>265</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cardImage_9">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>540</y>
      <width>190</width>
      <height>265</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="cardButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cardButton_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>290</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cardButton_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>560</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cardButton_4">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cardButton_5">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>290</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cardButton_6">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>560</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cardButton_7">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>20</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cardButton_8">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>560</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cardButton_9">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>290</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mtg_draft.qrc"/>
 </resources>
 <connections/>
</ui>
